src/frontend/src/pages/GanttPage/GanttChartComponents/GanttToolTip.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Typography } from '@mui/material';
+import { User } from 'shared';
+import { fullNamePipe } from '../../../utils/pipes';
+
+interface GanttToolTipProps {
+  xCoordinate: number;
+  yCoordinate: number;
+  title: string;
+  startDate: Date;
+  endDate: Date;
+  color?: string;
+  projectLead?: User;
+  projectManager?: User;
+}
+
+const GanttToolTip: React.FC<GanttToolTipProps> = ({
+  xCoordinate,
+  yCoordinate,
+  title,
+  startDate,
+  endDate,
+  projectLead,
+  projectManager
+}) => (
+  <Box
```

Comment:
Could we try making this translucent? I think the old style is a bit jarring with the black text

Reply:
yup which part should i make translucent? The background or font color

Reply:
The background

Reply:
It looked a little weird trasnparent so i made it like this, how does this look? 
<img width="1013" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/76594216/08f35a6e-6d5f-4189-85b0-65a554bb3fa5">

Reply:
can u send a pic of the transparent